ocfweb/stats/job_frequency.py

```diff
@@ -0,0 +1,106 @@
+import urllib.parse
+from collections import defaultdict
+from datetime import date
+from datetime import datetime
+
+import numpy as np
+import ocflib.printing.quota as quota
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse
+from django.shortcuts import redirect
+from matplotlib.figure import Figure
+from matplotlib.ticker import MaxNLocator
+
+from ocfweb.caching import periodic
+from ocfweb.component.graph import plot_to_image_bytes
+
+
+def pyday_to_sqlday(pyday):
+    """Converting weekday index from python to mysql."""
+    return (pyday + 1) % 7 + 1
+
+
+def get_jobs(sql='SELECT * FROM `public_jobs`'):
+    """Get the publicjobs database to python"""
+    conn = quota.get_connection(db='ocfprinting')
+    with conn.cursor() as curs:
+        curs.execute(sql)
+        return curs.fetchall()
+
+
+@periodic(60)
```

Comment:
Even though I think jobs are issued much much more frequent, we don't really have to check it that often(or at all). I guess 30 minutes is alright.